keyboards/keychron/k6/white/config.h

```diff
@@ -67,6 +62,19 @@
 /* Enable NKRO by default */ 
 #define FORCE_NKRO
 
+/* Enable led matrix effects */
+#define ENABLE_LED_MATRIX_ALPHAS_MODS
+#define ENABLE_LED_MATRIX_BREATHING
+#define ENABLE_LED_MATRIX_BAND
+#define ENABLE_LED_MATRIX_BAND_PINWHEEL
+#define ENABLE_LED_MATRIX_BAND_SPIRAL
+#define ENABLE_LED_MATRIX_CYCLE_LEFT_RIGHT
+#define ENABLE_LED_MATRIX_CYCLE_UP_DOWN
+#define ENABLE_LED_MATRIX_CYCLE_OUT_IN
+#define ENABLE_LED_MATRIX_DUAL_BEACON
+#define ENABLE_LED_MATRIX_WAVE_LEFT_RIGHT
+#define ENABLE_LED_MATRIX_WAVE_UP_DOWN
+
 /* Disable the following animation because they are not interesting in monochrome */
```

Comment:
the rgb effects disable list can go away on white. Also for the white LED driver, PWM channels must be enabled explicitely due to RAM issues. See https://github.com/SonixQMK/qmk_firmware/blob/sn32_master/keyboards/keychron/c1/white/config.h#L53-L75